src/Employer/Employer.Web/wwwroot/javascripts/application.js

```diff
@@ -174,6 +182,48 @@ function inViewport($el) {
     return Math.max(0, t > 0 ? Math.min(elH, wH - t) : Math.min(b, wH));
 }
 
+function initializeEditor(selector) {
+    tinymce.init({
+        element_format: 'html',
+        apply_source_formatting: true,
+        menubar: false,
+        plugins: 'lists',
+        selector: selector,
+        statusbar: false,
+        toolbar: 'bullist',
+        content_style: ".mce-content-body {font-size:19px;font-family:nta,Arial,sans-serif;}",
+        setup: function (tinyMceEditor) {
+            var element = tinyMceEditor.getElement();
+            
+            tinyMceEditor.on('keyup',
+                function (e) {
+                    setEditorMaxLength(element, tinyMceEditor);
+                });
+            tinyMceEditor.on('focus',
+                function (e) {
+                    tinyMceEditor.editorContainer.classList.add("editor-focus");
+                    console.log("focus");
+                });
+            tinyMceEditor.on('blur',
+                function (e) {
+                    tinyMceEditor.editorContainer.classList.remove("editor-focus");
+                    console.log("blur");
```

Comment:
Was this logging just for debugging purposes?

Reply:
👍